Hien/html 1

hoangtu7sao opened the pull request.

hoangtu7sao committed:
completed header layout

hoangtu7sao committed:
in progress footer layout

hoangtu7sao committed:
Completed page 5 layout

hoangtu7sao committed:
Completed page 6 layout

thiendaopk1 commented on pages/header.html:
```diff
@@ -0,0 +1,31 @@
+                    <path d="M12 11.4761C14.2091 11.4761 16 9.68521 16 7.47607C16 5.26694 14.2091 3.47607 12 3.47607C9.79086 3.47607 8 5.26694 8 7.47607C8 9.68521 9.79086 11.4761 12 11.4761Z" stroke="#1D1D1D" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                    </svg>
+                </td>
+                <td><svg width="33" height="40" viewBox="0 0 33 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
mấy cái icon em nên dùng thẻ img rồi nhúng cái src vào nha

thiendaopk1 commented on pages/cart.html:
```diff
@@ -0,0 +1,51 @@
+        <div class="coupon">
+            <h1>Cart total</h1>
+            <div>
+                <td>subtotal</td>
```
<td> thì phải đi chung với table, chỗ này em dùng thẻ span hoặc p nha, ưu tiên dùng span á

hoangtu7sao committed:
fix footer

hoangtu7sao committed:
fix header

hoangtu7sao committed:
completed page 7

hoangtu7sao committed:
inprogress fix page 5

hoangtu7sao committed:
move icons and change image name

hoangtu7sao committed:
add css iframe,footer,header,checkout page

hoangtu7sao committed:
add iframe header,footer.
change div(line11) to nav(line13)
add class page-link and page-present on line 14,15
add button PLACE ORDER line 72

hoangtu7sao committed:
add body for footer

hoangtu7sao committed:
add body for header

hoangtu7sao committed:
change nav to div line 11
change td to a on line18 ->21
change class page to page map line 17
add class line 13,14,24,44

hoangtu7sao committed:
delete because wrong way to name

hoangtu7sao committed:
add grid system

hoangtu7sao committed:
change css footer, header
delete checkout

hoangtu7sao committed:
add grid system to header and footer

hoangtu7sao committed:
Completed css page 7
just delete a comment in grid.css

hoangtu7sao committed:
Inprogress css page 6
completed abt 90%

hoangtu7sao committed:
delete wrong css in style.css

hoangtu7sao committed:
add css page 6

hoangtu7sao committed:
Inprogress css page 5

hoangtu7sao committed:
change media footer iframe

hoangtu7sao committed:
css footer iframe

hoangtu7sao committed:
Add class product cost

hoangtu7sao committed:
add bottom class product-name

hoangtu7sao committed:
css class order-info-area
